.github/workflows/publish.yml

```diff
@@ -28,14 +28,21 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install build
+        python -m pip install twine
 
     - name: Build packages
       run: python -m build
 
+    - name: Check metadata verification
+      run: python -m twine check dist/*
+
     # only publish distribution to PyPI for tagged commits
     - name: Publish distribution to PyPI
       if: startsWith(github.ref, 'refs/tags/v')
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
+        # do not run twine check by default
+        # to avoid duplicated output
+        verify-metadata: false
```

Comment:
I don't mind the duplicated output.  Maybe best to leave `verify-metadata` enabled, just to be cautious?

Reply:
I doubt the publishing action changes anything regarding that, but it's true that they may update those tools more often if new tools appear. Committed this change.